app/controllers/admin/book_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Admin::BookController < Admin::BaseController
+  before_action :find_book, except: :index
+
+  def show
+    @arr_author = BookAuthorship.author_id_arr(@book.id)
```

Comment:
```suggestion
    @arr_author = @book.book_authorships
```
ri nhé, cơ mà tên biến sửa lại nhé, thay vì aray thì tên là số nhìu là dc

Reply:
Dạ chị